Fix/update extending tina docs (#1293)
tinacms/tinacms.org: content/docs/extending-tina/advanced/overview.md

---

> title: Overview

Reviewer: ```suggestion
title: Advanced Usage
```

id: '/docs/advanced/advanced/overview'
next: /docs/extending-tina/advanced/extending-field-plugin
prev: /docs/extending-tina/edit-state
---

{{ WarningCallout text="This is only needed for advanced usage and special circumstances. Please refer to our [main docs](/docs/extending-tina/overview)  for most use case's" }}

> In some cases (for performed reasons or if you want to separate UI concerns from the schema), it might be necessary to use the underlying plugin system to do validation, custom components, etc. This is not our recommended path but it is possible and supported.

Reviewer: ```suggestion
In some cases (for performance reasons or if you want to separate UI concerns from the schema), it might be necessary to use the underlying plugin system to do validation, custom components, etc. This is not a recommended approach, but is possible and supported.
```
